src/utils/createTestProps.ts

```diff
@@ -0,0 +1,6 @@
+export default function createTestProps(testID: string) {
+  return {
+    testID,
+    accessibilityLabel: testID
```

Comment:
iss it necessary to hijack accessibility for testing?

Reply:
Not with `detox` but `appium` required it.